crates/bevy_asset/src/io/file/mod.rs

```diff
@@ -65,15 +65,14 @@ impl FileAssetReader {
     }
 }
 
+/// A writer for the local filesystem.
```

Comment:
This isn't that useful. Maybe you could add a little more? What's it writing and why do we need it?